### ISSUE TITLE
Vis code freezes

### ISSUE THREAD

**Issue by MartinAskestad** [Microsoft/vscode#52595]
Issue Type: <b>Performance Issue</b>

Visual Studio Code will freeze, usually when I switch back to the application from either the web browser or another program. It will also sometimes freeze when I'm just using the program, scrolling through files and such. Switching to another program seems to help, while VS Code is in the background it will do the things I've tried such as scrolling or opening another file and so on. Switching back to VS Code is the active program, it freezes again.

I've uninstalled every extension and removed all my user setting. The problem still persists. It's a angular 5 project created using angular-cli.

VS Code version: Code 1.24.1 (24f62626b222e9a8313213fb64b10d741a326288, 2018-06-13T17:51:32.889Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7600U CPU @ 2.80GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.83GB (8.53GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	    96	 19508	code main
    0	    75	  3792	   shared-process
    0	   276	  6704	   window (left-navigation.component.scss - Digital-Onboarding-Corporate - Visual Studio Code)
    0	    43	  6048	     searchService
    0	    72	 11132	     extensionHost
    0	    40	  2528	       electron_node jsonServerMain.js 
    0	   138	  6664	       electron_node electronForkStart.js tsserver.js 
    0	    36	 19488	         electron_node typingsInstaller.js tsserver.js typesMap.js 
    0	    56	 12588	       electron_node htmlServerMain.js 
    0	    38	 13160	       electron_node cssServerMain.js 
    0	    14	 22088	     watcherService 
    0	     9	 10300	       console-window-host (Windows internal process)
    0	   127	 11484	   gpu-process
    1	    71	 14872	   shared-process
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (left-navigation.component.scss - Digital-Onboarding-Corporate - Visual Studio Code)
|    Folder (Digital-Onboarding-Corporate): 240 files
|      File types: ts(170) html(30) scss(16) json(13) js(4) gitignore(1)
|                  prettierrc(1) png(1) md(1) config(1)
|      Conf files: package.json(1) tsconfig.json(1) tslint.json(1)
|                  launch.json(1);
```

</details>
Extensions: none

**Comment by chrmarti** [comment]
What does `code --status` print when VS Code is already frozen?

**Comment by jrieken** [comment]
Can you follow this guide: https://github.com/Microsoft/vscode/wiki/Performance-Issues#visual-studio-code-is-consuming-a-lot-of-cpu

**Comment by MartinAskestad** [comment]
I can't seem to reproduce it much anymore. I uninstalled all my extensions, reloaded the window, the problem was still there. Then all of a sudden a message told me that "extensions has changed on disk, reload the window" or something similar. So I did. And the problem has occured a lot less now.

It did seem to occur when I reinstalled and ran the chrome debugger extension, the window went completley black and then it came back again. The terminal was still black and the code-minimap to the right was black until I closed the current file and terminal.

Could it be related to some hardware acceleration problem? Can I toggle hardware acceleration myself to test it out?
